### DISCUSSION TITLE
[General] Bridge Withdraw L2 to L1

### DISCUSSION THREAD

**Discussion by Graypoint-dev** [zkSync-Community-Hub/zksync-developers#289] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement.

We deposited and withdrawn our ERC20 tokens at txSync.io.
widthdraw has been completed... Tokens withdrawn from L1 network Ethereum
Even after several days, the deposit has not been received.
https://explorer.zksync.io/tx/0x083e9ecf67794c2e1ccd855c7d1c26c836047c89a966af5dc84ae4ac776ac43e
Why doesn't withdraw work properly?

### What is the specific issue or error you're encountering?

ERC20 tokens are not being withdrawn.
Our ERC20 token is not listed at https://explorer.zksync.io/tokens.

### Can you share the error messages or logs you're receiving, if any?

None

### Have you made any recent changes to the contract before encountering this issue?

None

### Are there any external libraries or contracts that your contract interacts with?

https://portal.txsync.io/

### Can you provide the relevant portions of your contract code where the issue is occurring?

None

### Have you tried to isolate the problem, and if so, what were the results?

None

### What steps have you already taken to try to resolve the issue?

Tried several times.

**Comment by vasyl-ivanchuk** [comment]
Hi @Graypoint-dev 👋 , we will check it on our end and follow up.

**Reply by vasyl-ivanchuk** [reply, marked as answer]
@Graypoint-dev , this token wasn’t auto detected by our finalizer services. We are aiming to improve auto detection but don’t have an ETA for it. For now withdrawals must be finalized manually. We have already finalized the withdrawals you mentioned:
https://explorer.zksync.io/tx/0x083e9ecf67794c2e1ccd855c7d1c26c836047c89a966af5dc84ae4ac776ac43e
https://explorer.zksync.io/tx/0x4147522d3d74d7d7990e15dcba18688724e5c2125fa800ebbd4746b111604ff1

**Comment by hinataa01** [comment]
thank u

**Comment by Graypoint-dev** [comment]
The details requested above were confirmed on Ethereum 4 hours ago.
But...
The following details are withdrawals for coins from our service, and have not yet been confirmed on Ethereum.
Isn’t withdrawal automatic now?

https://explorer.zksync.io/tx/0xc00f34bd25d258ca8a3bc64cce61dbdd50b8d6270619df98e55ce6cc7f85078a
https://explorer.zksync.io/tx/0x9019c47d902fa1f9967130389d379ed80b09255ae42be2bfa47ad42e0646ee9d

**Reply by vasyl-ivanchuk** [reply]
@Graypoint-dev similar to the previous token, this one was not auto detected by our finalizer as well. These withdrawals are finalized now.

**Reply by Graypoint-dev** [reply]
The action taken has been confirmed.
Thank you ^^
